apps/docs/src/app/platform/component-docs/platform-button/platform-button-examples/platform-button-sizes-example.component.html

```diff
@@ -1,2 +1,2 @@
 <fdp-button>Normal Button</fdp-button>
-<fdp-button [compact]="true">Compact Button</fdp-button>
+<fdp-button [contentDensity]="'compact'">Compact Button</fdp-button>
```

Comment:
Can we consistent with core and use the same naming?

Reply:
With respect to platform component consistency will be having it as content density.

Reply:
With respect to platform component consistency will be having it as content density.